acceptance/catalog_services_test.go

```diff
@@ -15,9 +15,14 @@ var _ = Describe("Catalog Services", func() {
 	})
 
 	Describe("create-service", func() {
-		It("creates a catalog based service", func() {
+		It("creates a catalog based service, with waiting", func() {
 			makeCatalogService(serviceName)
 		})
+
+		It("creates a catalog based service, without waiting", func() {
+			makeCatalogServiceDontWait(serviceName)
```

Comment:
Can't this test run into a race condition with the `AfterEach` cleanup?

Reply:
No. The `without waiting` is for the core `create-service` command. The wrapper then waits explicitly on its own, using the `service command`.